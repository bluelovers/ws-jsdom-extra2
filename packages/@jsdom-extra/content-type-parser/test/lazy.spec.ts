import { basename, extname } from 'path';
import { parseContentType, parseContentTypeCharset } from '../index';

test(`test`, () =>
{

	expect(parseContentType(`Text/HTML;Charset="utf-8"`)).toMatchSnapshot();

	expect(parseContentTypeCharset(`Text/HTML;Charset="utf-8"`)).toMatchSnapshot();

});
