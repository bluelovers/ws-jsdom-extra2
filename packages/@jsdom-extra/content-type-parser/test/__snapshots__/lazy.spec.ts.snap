// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
MIMEType {
  "_parameters": MIMETypeParameters {
    "_map": Map {
      "charset" => "utf-8",
    },
  },
  "_subtype": "html",
  "_type": "text",
}
`;

exports[`test 2`] = `"utf-8"`;
